### PR TITLE
Ticket 49701 - Test coverage for Quick Edit UI for taxonomies with show_ui=false and show_in_quick_edit=true

### PR DESCRIPTION
**Pull request by figureone**
Added tests for hierarchical and nonhierarchical taxonomies with `show_ui = false` and `show_in_quick_edit = true`.

Trac ticket: https://core.trac.wordpress.org/ticket/49701

### CODE REVIEW

**Comment by peterwilsoncc** [comment]
I've re-targeted this to `trunk` to account for the new default branch name and merged in an update to get the tests running.

File: tests/phpunit/tests/taxonomy.php [hunk 1]
```diff
@@ -238,6 +238,54 @@ public function test_register_taxonomy_fires_registered_actions() {
 		$this->assertSame( 3, $action->get_call_count() );
 	}
 
+	/**
+	 * @ticket 49701
```
Comment (costdev) [inline tests/phpunit/tests/taxonomy.php:242]:
```suggestion
	 * @ticket 49701
	 *
	 * @covers ::get_inline_data
```

File: tests/phpunit/tests/taxonomy.php [hunk 2]
```diff
@@ -238,6 +238,54 @@ public function test_register_taxonomy_fires_registered_actions() {
...
+		wp_set_object_terms( $post->ID, $term['term_id'], 'wptests_tax_1' );
+
+		// Test get_inline_data() has post_category div containing assigned term.
+		wp_set_current_user( self::factory()->user->create( array( 'role' => 'editor' ) ) );
+		get_inline_data( $post );
+		$this->expectOutputRegex( '/<div class="post_category" id="wptests_tax_1_' . $post->ID . '">' . $term['term_id'] . '<\/div>/' );
+	}
+
+	/**
+	 * @ticket 49701
```
Comment (costdev) [inline tests/phpunit/tests/taxonomy.php:268]:
```suggestion
	 * @ticket 49701
	 *
	 * @covers ::get_inline_data
```

**Review by costdev — CHANGES_REQUESTED** [review]
Just needs `@covers` annotations and then I think this one is good to go. 👍

**Comment by audrasjb** [comment]
I committed changes suggested by @costdev to the PR :)
Now leaving this to @SergeyBiryukov for final commit :)

**Comment by peterwilsoncc** [comment]
These tests were merged in https://core.trac.wordpress.org/changeset/53368, thank you!

**Pull request closed without merging**